flink-connectors/flink-connector-files/src/main/java/org/apache/flink/connector/file/table/FileSystemCommitter.java

```diff
@@ -83,7 +87,17 @@ public void commitPartitions() throws Exception {
         List<Path> taskPaths = listTaskTemporaryPaths(fs, tmpPath);
 
         try (PartitionLoader loader =
-                new PartitionLoader(overwrite, fs, metaStoreFactory, isToLocal)) {
+                new PartitionLoader(
+                        overwrite,
+                        fs,
+                        metaStoreFactory,
+                        isToLocal,
+                        flinkConfig.get(
```

Comment:
Pass the `flinkconf` to `PartitionLoader` ?

Reply:
Based on your suggestions, I pass `PartitionCommitPolicy` to `PartitionLoader`,  when `PartitionCommitPolicy` is not null, apply this policy.